app/boards/arm/bluemicro52840_v1/Kconfig.defconfig

```diff
@@ -0,0 +1,30 @@
+# Electronut Labs Papyr board configuration
```

Comment:
This seems out of place... I see it in the nice!nano def too. Oops?

Reply:
Yeah, this should be fixed, and we should fix in the nano board too.